emails/apps.py

```diff
@@ -30,8 +30,21 @@ def __init__(self, app_name, app_module):
         )
         with open(badwords_file_path, 'r') as badwords_file:
             for word in badwords_file:
+                if len(word.strip()) > 0 and word.strip()[0] == "#":
+                    continue
                 badwords.append(word.strip())
         self.badwords = badwords
 
+        blocklist = []
+        blocklist_file_path = os.path.join(
+            settings.BASE_DIR, 'emails', 'blocklist.txt'
+        )
+        with open(blocklist_file_path, 'r') as blocklist_file:
+            for word in blocklist_file:
+                if len(word.strip()) > 0 and word.strip()[0] == "#":
+                    continue
+                blocklist.append(word.strip())
+        self.blocklist = blocklist
+
```

Comment:
Might be able to reduce some code duplication here (note: I didn't test this, just an idea for it):
```
diff --git a/emails/apps.py b/emails/apps.py
index 2cd9cbc..4eea17e 100644
--- a/emails/apps.py
+++ b/emails/apps.py
@@ -22,29 +22,24 @@ class EmailsConfig(AppConfig):
         except Exception:
             logger.exception("exception during SES connect")
 
-        badwords = []
         # badwords file from:
         # https://www.cs.cmu.edu/~biglou/resources/bad-words.txt
-        badwords_file_path = os.path.join(
-            settings.BASE_DIR, 'emails', 'badwords.txt'
-        )
-        with open(badwords_file_path, 'r') as badwords_file:
-            for word in badwords_file:
-                if len(word.strip()) > 0 and word.strip()[0] == "#":
-                    continue
-                badwords.append(word.strip())
-        self.badwords = badwords
+        self.badwords = self._load_terms('badwords.txt')
+        # blocklist file from:
+        # https://github.com/michaldudek/subdomain-blacklist
+        self.blocklist = self._load_terms('blocklist.txt')
+
 
-        blocklist = []
-        blocklist_file_path = os.path.join(
-            settings.BASE_DIR, 'emails', 'blocklist.txt'
-        )
-        with open(blocklist_file_path, 'r') as blocklist_file:
-            for word in blocklist_file:
-                if len(word.strip()) > 0 and word.strip()[0] == "#":
+    def _load_terms(self, filename):
+        terms = []
+        terms_file_path = os.path.join(settings.BASE_DIR, 'emails', filename)
+        with open(terms_file_path, 'r') as terms_file:
+            for term in terms_file:
+                if len(term.strip()) > 0 and term.strip()[0] == "#":
                     continue
-                blocklist.append(word.strip())
-        self.blocklist = blocklist
+                terms.append(term.strip())
+        return terms
+
 
     def ready(self):
         import emails.signals
```

Reply:
Heh, I was being a bit lazy since it would cost me more time than I'd like to look up how to do this in Python, so thanks for the helpful pointer :) e9b4016